### DISCUSSION TITLE
New table without any sync

### DISCUSSION THREAD

**Discussion by maxrodro** [Mimetis/Dotmim.Sync#519] [Q&A] [answered]
Hi, I made a table on the server that I want to be on the client but I don't want it to have any type of Bidirectional, Upload or Download synchronization, is it possible to do it?

If I just add it to the SyncSetup, it throws me this error

`Microsoft.Data.SqlClient.SqlException (0x80131904): Change tracking is not enabled on table 'Logs'.\r\n   `

**I do not require any type of change tracking**

**Comment by Mimetis** [comment, marked as answer]
You just want to recreate the table on the client ?
If yes, do not use DMS for that, just create the table on the client with any ORM or SQL Script.

**Reply by maxrodro** [reply]
I want to have it on both sides, but each one independent of the data

**Reply by Mimetis** [reply]
Well, not sure it's something that should be handled by `DMS`, but we can think differently.
Could it be an idea to have a new option in `SyncDirection` that have a new value of `None` ?


That being said, if we have this new value `SyncDirection.None`, it will implies that:
- Table structure will be replicated
- Stored procedures, triggers and tracking tables will be created (depending the provider)
- No datas will be replicated until the `SyncDirection` is changed

Is it the idea you have ?

**Reply by maxrodro** [reply]
Good morning @Mimetis 

It would be good if it **only replicates the table** nothing else, that there is no subject of changes.

SyncDirection.None -> Solve my requirement.

**Reply by Mimetis** [reply]
It's available on the last release `v0.7.3` !
